CDCgov/prime-reportstream
# update submission received event and delete v1 azure events

This PR deletes the v1 azure event data classes and improves the event emitted from the submission controller. It now includes queryParamters, httpMethod, and full request url. 

Test Steps:
1. `./gradlew bootRun` to make sure it compiles
2. run submissionController tests
3. run submissionControllerIntegration tests

## Changes
- update SubmissionReceivedEvent
- add retrieval and filtering for query parameters
- retrieve url and http method
- delete unused azure event data classes

## Checklist

### Testing

## Linked Issues
- Linked #13161
